pkg/docker/docker.css

```diff
@@ -628,3 +628,22 @@ table.drive-list td:nth-child(2) img {
     padding-left: 20px;
     padding-right: 20px;
 }
+
+#delete-image-confirmation-dialog-containers {
```

Comment:
It's generally best to avoid IDs in CSS when possible.

1. IDs raise specificity and cause problems (which isn't really a big deal here, thankfully)
2. Generally styles should be implemented in a more generic, reusable way (which might also not be a big deal here)

Cockpit has this problem all over its codebase.

It's not much of a concern in this code, but you (and everyone else on the team too) should be aware of it.